docs/include/conf/past.rst

```diff
@@ -9,8 +9,8 @@ North America
 - `Write the Docs North America, May 5-6 2014, Portland, Oregon`_
 - `Write the Docs, Apr 8-9 2013, Portland, Oregon`_
 
-.. _Write the Docs North America, May 22-24 2016, Portland, Oregon: http://www.writethedocs.org/conf/na/2016/
-.. _Write the Docs North America, May 17-19 2015, Portland, Oregon: http://www.writethedocs.org/conf/na/2015/
+.. _Write the Docs North America, May 22-24 2016, Portland, Oregon: /conf/na/2016/
```

Comment:
Hmm, for me it's clearer what it's linking to by using the absolute link (`/conf/...`), rather than a relative link that points to a file in a different directory. I can change it if you disagree though. :)